### ISSUE TITLE
Bad ARN format for ClinVar dataset

### ISSUE THREAD

**Issue by carvantes** [awslabs/open-data-registry#864]
Related to https://github.com/awslabs/open-data-registry/pull/815#discussion_r577061543

The clinvar study has a malformed ARN:

https://github.com/awslabs/open-data-registry/blob/945edf1ba8ba0bc06a113adc64e77569640f795d/datasets/clinvar.yaml#L17

I'd expect it to be `arn:aws:s3:::aws-roda-hcls-datalake/clinvar_summary_variants/`

I think that this is also breaking the website at https://registry.opendata.aws/clinvar/. There's an `undefined` in the AWS CLI command
```
AWS CLI Access (No AWS account required)
    aws s3 ls s3://undefined/ --no-sign-request
```
---
btw, Is there validation in place to prevent this kind of malformed data from being published? I see that there is a schema file with a regex for ARN, but somehow this bad ARN ended up being published
https://github.com/awslabs/open-data-registry/blob/95a8388117bc8111977edeb3f35e174e06ca4b8b/schema.yaml#L70-L72

I'm working on https://github.com/awslabs/service-workbench-on-aws and we consume the open data files. This bad ARN caught by surprise and is causing bugs. We are thinking on adding extra validation on our side but it's probably best for all your clients if you validate ARNs on your side.

**Comment by nguyen102** [comment]
Related
https://github.com/awslabs/open-data-registry/pull/815#discussion_r577066760

**Comment by zflamig** [comment]
Thanks! Fixed in #865
